package.json

```diff
@@ -38,6 +38,10 @@
     "eslint": "8.28.0",
     "eslint-config-next": "13.0.5",
     "formik": "2.4.1",
+    "fp-ts": "^2.16.1",
+    "io-fs": "^1.0.0",
+    "io-ts": "^2.2.21",
+    "jsdom": "^23.0.1",
```

Comment:
This should be moved in `devDependencies` as it's only for tests

Reply:
rmoved.